turbo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "tasks": {
+    "compile": {
+      "dependsOn": ["^compile"],
```

Comment:
is this right? Does this mean that we're going to require the Rust compiler to be built before we build the JS?

Reply:
The arrow ^ has the same meaning as pnpm -r - dependencies of each package need to be compiled first

Reply:
Okay TIL